frontend/src/pages/NewRunV2.test.tsx

```diff
@@ -113,7 +113,7 @@ describe('NewRunV2', () => {
       pipeline_id: ORIGINAL_TEST_PIPELINE_ID,
       pipeline_version_id: ORIGINAL_TEST_PIPELINE_VERSION_ID,
     },
-    runtime_config: { parameters: { intParam: 123 } },
+    runtime_config: { parameters: { intParam: 123 }, pipeline_root: 'hello/world' },
```

Comment:
nit: probably choose a different dummy value for pipeline root, so it's more clear what type of value is expected. e.g. `gs://dummy_pipeline_root`